api_docs/unmerged.d/ZF-4c2e41.md

```diff
@@ -0,0 +1,8 @@
+* [`GET /users/me/subscriptions`](/api/get-subscriptions),
+  [`GET /streams`](/api/get-streams), [`GET /events`](/api/get-events),
+  [`POST /register`](/api/register-queue): Added new `empty_topic_only`
+  configuration to `topics_policy` field in Stream and Subscription objects.
+* [`POST /users/me/subscriptions`](/api/subscribe),
+  [`PATCH /streams/{stream_id}`](/api/update-stream): Added new
+  `empty_topic_only` configuration to `topics_policy` to support disabling
+  of topics in the channels.
```

Comment:
`configuration` is a vague word. In API docs, we try to be precise -- so here, "option" would be a lot more specific.

"disabling of topics" is also weird grammar; "to support channels with topics disabled." would be a lot better ending for that sentence.